src/IndexedDbProvider.ts

```diff
@@ -435,7 +436,7 @@ class IndexedDbStore implements NoSqlProvider.DbStore {
                 if (this._fakeComplicatedKeys) {
                     // Fill out any compound-key indexes
                     if (NoSqlProviderUtils.isCompoundKeyPath(this._schema.primaryKeyPath)) {
-                        item['nsp_pk'] = NoSqlProviderUtils.getSerializedKeyForKeypath(item, this._schema.primaryKeyPath);
+                        (item as any)['nsp_pk'] = NoSqlProviderUtils.getSerializedKeyForKeypath(item, this._schema.primaryKeyPath);
```

Comment:
isn't this mutating the input object? (not a behavioral change but still seems backwards)

Reply:
You're not wrong ... we didn't want to clone it just to change this field.  Technically we should maybe delete the nsp_pk field from it afterward...